src/Shared/Extensions/PackageUrlExtension.cs

```diff
@@ -40,6 +41,26 @@ public static bool HasNamespace(this PackageURL packageUrl)
         return packageUrl.Namespace.IsNotBlank();
     }
     
+    /// <summary>
+    /// We want npm package's namespace to be prefixed with "%40" the percent code of "@".
+    /// </summary>
+    /// <param name="packageUrl">The <see cref="PackageURL"/> to get a formatted namespace for.</param>
+    /// <returns>The formatted namespace of <paramref name="packageUrl"/>.</returns>
+    public static string GetNamespaceFormatted(this PackageURL packageUrl)
```

Comment:
For my learning, does this logic apply only to npm or are their other package managers using `@` in their namespace. It'll be helpful to put an example of a formatted namespace in the comments to better understand the logic here.

Reply:
As far as I'm aware, yes it is just for NPM. I can add a remark saying that if the `packageUrl` isn't `npm` then it just returns the namespace as is.